src/app/story/directives/basic.component.ts

```diff
@@ -48,6 +48,17 @@ import { StoryModel, TabService } from '../../shared';
       tiptext="More is better!" >
         <div class="fancy-hint">A comma-separated list of tags relevant to the content of your piece.</div>
       </publish-fancy-field>
+      
+      <publish-fancy-field label="Release Date">
+        <div class="fancy-hint">If the release date is not set manually,
+          the current date will be used for publishing.</div>
+        <publish-datepicker 
```

Comment:
That CSS is loaded by the angular-cli, so if you don't restart the app, it doesn't get loaded.